warehouse/views.py

```diff
@@ -151,26 +151,6 @@ def forbidden(exc, request):
             )
             return HTTPSeeOther(url)
 
-        # If the forbidden error is because the user doesn't have 2FA enabled, we'll
-        # redirect them to the 2FA page
-        if exc.result.reason in {
-            "owners_require_2fa",
-            "pypi_mandates_2fa",
-            "manage_2fa_required",
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏With this line removed, how will a user without 2FA know what to do next? This reason is set in https://github.com/pypi/warehouse/pull/15131/files#diff-60b1172fcc3ed0f44469ecfff8ba1b5021c2fca3686eb4837522291a722ef624L358-L361

Reply:
Thanks, good catch: f16fd1c